.github/workflows/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+name: Docker Test
+
+on: [pull_request, workflow_dispatch]
+jobs:
+  test:
+    name: Run test suite
+    runs-on: ubuntu-latest
+    env:
+      DATABASE_URL: ${{ secrets.DATABASE_URL }}
+      POSTGRES_USER: "kartik"
+      POSTGRES_PASSWORD: "123"
+      POSTGRES_DB: "notes_data"
+      COMPOSE_FILE: ./server/docker-compose.yml
+      DOCKER_USER: ${{ secrets.DOCKER_USER }}
+      DOCKER_PASS: ${{ secrets.DOCKER_PASS }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Login to DockerHub
+        run: docker login -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
Why are we logging to DockerHub again? Can we not use the `Dockerfile` from this repo?

Reply:
We need to login for Dockerfile to PULL images.